docs/src/create-apps/hooks/_index.md

```diff
@@ -75,185 +103,207 @@ During the `build` hook, the site isn't yet deployed and so there is no URL to u
+and [Next.js configuration](https://github.com/platformsh-templates/nextjs-drupal/blob/master/client/.platform.app.yaml)
 on GitHub.
 The following shows only the parts necessary for the hooks.
 
 ### Drupal
 
-```yaml {location="drupal/.platform.app.yaml"}
+```yaml {location="api/.platform.app.yaml"}
 # The name of this app. Must be unique within the project.
 name: 'drupal'
 
-# The runtime the application uses.
-type: 'php:7.4'
+# The runtime the app uses.
+type: 'php:8.1'
 
 dependencies:
     php:
         composer/composer: '^2'
 
-# The relationships of the application with services or other applications.
+# The relationships of the app with services or other apps.
 relationships:
     database: 'db:mysql'
     redis: 'cache:redis'
 
-# The hooks executed at various points in the lifecycle of the application.
+# The hooks executed at various points in the lifecycle of the app.
 hooks:
-    # The build hook runs after Composer to finish preparing your code.
-    build: |
-        set -e
-        bash install-redis.sh 4.3.0
-    # The deploy hook runs after your application has been deployed and started,
-    # but before it starts accepting requests
-    deploy: |
-        set -e
-        php ./drush/platformsh_generate_drush_yml.php
-        cd web
-        drush -y cache-rebuild
-        drush -y updatedb
-        drush -y config-import
+    deploy: !include
+      type: string
+      path: platformsh-scripts/hooks.deploy.sh
 
-# The size of the persistent disk of the application (in MB).
+# The size of the persistent disk of the app (in MB).
 disk: 2048
 
-# The 'mounts' describe writable, persistent filesystem mounts in the application.
+# The 'mounts' describe writable, persistent filesystem mounts in the app.
 mounts:
-    '/.drush':
+    /.drush:
         source: local
         source_path: 'drush'
+    /drush-backups:
+        source: local
+        source_path: 'drush-backups'
+    deploy:
+        source: service
+        service: files
+        source_path: deploy
 ```
 
-### Gatsby
+### Next.js
 
-```yaml {location="gatsby/.platform.app.yaml"}
-# The name of this application, which must be unique within the project.
-name: 'gatsby'
+```yaml {location="client/.platform.app.yaml"}
+# The name of this app, which must be unique within the project.
+name: 'nextjs'
 
-# The type key specifies the language and version for your application.
+# The type key specifies the language and version for your app.
 type: 'nodejs:14'
 
+dependencies:
+    nodejs:
+        yarn: "1.22.17"
+        pm2: "5.2.0"
+
+build:
+    flavor: none
+
 relationships:
-    drupal: 'drupal:http'
+    api: 'api:http'
 
 # The hooks that will be triggered when the package is deployed.
 hooks:
-    # Gatsby's build is delayed to the post_deploy hook, when Drupal is available for requests.
-    post_deploy: npm run build
+    build: |
+        set -e
+        yarn --frozen-lockfile
+
```

Comment:
:pick:
```suggestion
```

Reply:
It was there intentionally to split the sections. As above.